src/components/PersonItem.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+
+  const { name, sex, born, died, motherName, fatherName, mother, father } =
+    person;
+
+  return (
+    <tr
+      data-cy="person"
+      key={person.slug}
+      className={classNames({
+        'has-background-warning': person.slug === slug,
+      })}
+    >
+      <td>
+        <PersonLink person={person}>{name}</PersonLink>
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+
+      <td>
+        {mother && motherName ? (
+          <PersonLink person={mother}>{motherName}</PersonLink>
```

Comment:
```suggestion
          <PersonLink person={mother} />
```
Just pass a person object to `PersonLink`, and from this object, you can get a name and show it.
Fix in all places